misc/codegen/test/test_qlgen.py

```diff
@@ -861,25 +861,47 @@ def test_property_on_class_with_default_doc_name(generate_classes):
 
 def test_stub_on_class_with_ipa_from_class(generate_classes):
     assert generate_classes([
-        schema.Class("MyObject", ipa=schema.IpaInfo(from_class="A")),
+        schema.Class("MyObject", ipa=schema.IpaInfo(from_class="A"),
+                     properties=[schema.SingleProperty("foo", "bar")]),
     ]) == {
         "MyObject.qll": (a_ql_stub(name="MyObject", base_import=gen_import_prefix + "MyObject", ipa_accessors=[
             ql.IpaUnderlyingAccessor(argument="Entity", type="Raw::A", constructorparams=["result"]),
         ]),
-            a_ql_class(name="MyObject", final=True, ipa=True)),
+            a_ql_class(name="MyObject", final=True, properties=[
+                ql.Property(singular="Foo", type="bar", tablename="my_objects", synth=True,
+                            tableparams=["this", "result"], doc="foo of this my object"),
+            ])),
     }
 
 
 def test_stub_on_class_with_ipa_on_arguments(generate_classes):
     assert generate_classes([
-        schema.Class("MyObject", ipa=schema.IpaInfo(on_arguments={"base": "A", "index": "int", "label": "string"})),
+        schema.Class("MyObject", ipa=schema.IpaInfo(on_arguments={"base": "A", "index": "int", "label": "string"}),
+                     properties=[schema.SingleProperty("foo", "bar")]),
```

Comment:
Am I reading this correctly that when you add a simple `foo: bar` property, it becomes `synth=True` by default? Or is it because this class is declared as inheriting from class "A" and its `foo: bar | synth` property on line 575 of test_dbschemegen.py? Basically, I'm finding it hard to follow how the fixtures for this test are set up.

Reply:
Oh, right, I didn't realize that that's what `ipa=` was doing. Doing that renaming would indeed be clearer, yes. I thought that this PR did the complete renaming, but apparently it's backwards-compatible with `ipa`, so it's probably also fine as it is.